frontend/src/components/blocks/BlockDetails.tsx

```diff
@@ -82,9 +82,16 @@ const BlockDetails = ({ block }: Props) => {
                     }
                     imgLink="/static/images/icon-t-status.svg"
                     text={
-                      typeof context.finalTimestamp === "undefined"
+                      context.finalityStatus &&
+                      typeof context.finalityStatus
+                        .finalBlockTimestampNanosecond === "undefined"
```

Comment:
This should do the trick:

```suggestion
                      typeof context.finalityStatus?.finalBlockTimestampNanosecond === "undefined"
```